### PR TITLE
*: update description about getting support resources

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add methods about getting support.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/12356
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+如果你在使用 TiDB 的过程中遇到了问题，你可以通过以下方式向 PingCAP 或 TiDB 社区寻求帮助：
+
++ 从 PingCAP 获取支持（需要订阅 [TiDB 企业版](https://www.pingcap.com/tidb-enterprise/)）：
+
+    - [提交工单](support.pingcap.cn)
+
++ 从 TiDB 社区寻求帮助：
+
+    - [AskTUG 论坛](https://asktug.com/)
+    - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb) (在 #tidb 标签下提问)
```
Comment (ran-huang) [inline support.md:17]: removed slack from Chinese docs

File: support.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+如果你在使用 TiDB 的过程中遇到了问题，你可以通过以下方式向 PingCAP 或 TiDB 社区寻求帮助：
+
++ 从 PingCAP 获取支持（需要订阅 [TiDB 企业版](https://www.pingcap.com/tidb-enterprise/)）：
+
+    - [提交工单](https://support.pingcap.cn/)
+
++ 从 TiDB 社区寻求帮助：
+
+    - [AskTUG 论坛](https://asktug.com/)
+    - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb) (在 #tidb 标签下提问)
```
Comment (lilin90) [inline support.md:17]:
```suggestion
    - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb)（在 #tidb 标签下提问）
```

File: support.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: 支持资源
+summary: 在使用 TiDB 时遇到问题，如何获取支持。
+---
+
+# 支持资源
+
+如果你在使用 TiDB 的过程中遇到了问题，你可以通过以下方式向 PingCAP 或 TiDB 社区寻求帮助：
+
++ 从 PingCAP 获取支持（需要订阅 [TiDB 企业版](https://www.pingcap.com/tidb-enterprise/)）：
```
Comment (lilin90) [inline support.md:10]:
```suggestion
+ 从 PingCAP 获取支持（需要订阅 [TiDB 企业版](https://cn.pingcap.com/product-enterprise/)）：
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**